### PR TITLE
Creating Leovaunt Halo RPG

### PR DESCRIPTION
**Pull request by jweweiler**

### CODE REVIEW

File: Leovaunt Halo RPG/sheet.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"html": "halo.html",
+	"css": "halo.css",
+	"authors": "Jordan Weiler",
+	"roll20userid": "2510115",
+	"patreon": "https://www.patreon.com/leovauntgamedesign",
+	"preview": "halo.png",
+	"instructions": "Lines seperate each section. Roll buttons are located next to their associated stats. Find the game at the patreon page or at 'https://www.stellarrealmsgames.com/'. Enjoy! **343 Industries owns all rights to Halo.**"
```
Comment (Anduh) [inline Leovaunt Halo RPG/sheet.json]:
```suggestion
	"instructions": "Lines seperate each section. Roll buttons are located next to their associated stats. Find the game at the [patreon page](https://www.patreon.com/leovauntgamedesign) or at [Stellar Realms Games](https://www.stellarrealmsgames.com/). Enjoy! **343 Industries owns all rights to Halo.**"
```

Comment (jweweiler) [reply Leovaunt Halo RPG/sheet.json]: I wasn't aware of that, but thanks. That's pretty nice to have. Is it already changed, or should I personally make the change?

Comment (Anduh) [reply Leovaunt Halo RPG/sheet.json]: This is just a suggestion I made, you need to click on some button like **'Approve'** or **Apply changes** for it to become a part of your PR.

Comment (jweweiler) [reply Leovaunt Halo RPG/sheet.json]: I've tried looking everywhere and following the link you provided, but I don't see any way for me to approve the suggestion. Based on that link, I think I would need write access to the repository to do so. Regardless, I could probably update that manually later.

> You can apply suggested changes directly in a pull request **if you have write access** to the repository and are the author of or assigned to the pull request

**Comment by clevett** [comment]
@jweweiler 

You've got CSS in your HTML file. You'll need to fix this and submit again.

**Pull request merged**